### PR TITLE
Add author info for Kim Goh

### PR DESCRIPTION
**Pull request by lcnja**
The first push, add author info.

### CODE REVIEW

**Comment by lcnja** [comment]
Please review,Thx

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # yacht
 A basic webapp with authorization management
 
-##### @authors brandonccx lovelyun
+##### @authors brandonccx lovelyun lcnja
```
Comment (brandonccx) [inline README.md]: It's better to type your name _Kim Goh_ ?

**Comment by lcnja** [comment]
@brandonccx 
Yeah,I will modify and resubmit

**Comment by lcnja** [comment]
@brandonccx @lovelyun 
Please review,Thx

**Pull request merged**